app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/registration/RegisterWithEoriAndIdController.scala

```diff
@@ -267,10 +273,11 @@ class RegisterWithEoriAndIdController @Inject() (
       }
   }
 
-  private def onRegistrationPassCheckSubscriptionStatus(service: Service, idType: String, id: String)(implicit
+  private def onRegistrationPassCheckSubscriptionStatus(idType: String, id: String)(implicit
     request: Request[AnyContent],
     loggedInUser: LoggedInUserWithEnrolments,
-    hc: HeaderCarrier
+    hc: HeaderCarrier,
+    service: Service
```

Comment:
It has been named as originationgService in most of the places, can you make sure you rename all these service vals into originatingservice so there is consistency?

Reply:
As discussed, It's named as service already in the controller and didn't rename since it is moved into implicit scope.